app/process/watcher/prometheus/watcher.go

```diff
@@ -241,6 +217,10 @@ func getMetricData(
 }
 
 func (pw Watcher) setMetrics() {
+	if !pw.prometheusService.GetIsMonitoringEnabled() {
```

Comment:
I think we have the same case here as on row [71](https://github.com/LimeChain/hashport-validator/blob/feature/dynamic-min-fungible-fees/app/process/watcher/prometheus/watcher.go#L71) -the if in [main.go](https://github.com/LimeChain/hashport-validator/blob/feature/dynamic-min-fungible-fees/cmd/main.go#L103)